swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -162,7 +164,11 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
       adapter?.setInternetResourceEnabled(value)
     case .signOut:
       Task {
-        try await Token.delete()
+        do {
```

Comment:
I wrapped this one too and updated it to the use the Sentry error logging.

Reply:
We can't do this one level further up?

Reply:
Hm yeah, that would probably be a bit cleaner. We may want to actually use a `DispatchQueue` here which would guarantee ordering. I'll think a bit more about it.

Reply:
Ok, I don't think we care too much about ordering here. I've simply removed the unnecessary `Task` spawns inside the `PacketTunnelProvider` IPC handlers.

The only call that may take a while is `WrappedSession.connect` in Adapter since we perform a DNS lookup there and establish the websocket. It's ok to block the startTunnel thread on that since we depend on its return value to determine whether the tunnel came up or not.